scripts/freeze

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh -e
+
+# Use the Python executable provided from the `-p` option, or a default.
+[ "$1" = "-p" ] && PYTHON=$2 || PYTHON="python3"
+
+VENV=$(mktemp -d ./venv-XXXXXXXXXX)
+
+set -x
+
+"${PYTHON}" -m venv "${VENV}"
+PIP="${VENV}/bin/pip"
+
+"${PIP}" install -U "pip >= 20.2" setuptools wheel
+"${PIP}" install -r requirements.in
+
+# Argument "--all" is needed to pin "wheel"
```

Comment:
Okay, tho' we're never installing from the pinned version of `wheel`, since we always preemptively use:

`pip install -U pip setuptools wheel`

So I don't think we really need/want `--all` here.